core/commands/object/object.go

```diff
@@ -509,6 +509,13 @@ Available templates:
 			res.SetError(err, cmdkit.ErrNormal)
 			return
 		}
+
+		err = n.Providers.Provide(k)
+		if err != nil {
```

Comment:
As you have done in other files, these error checks should be condensed to a single line. This will keep `err` in the scope of the if statement.